src/grad.jl

```diff
@@ -5,7 +5,7 @@ export grad, jacobian, jvp, j′vp, to_vec
 
 Approximate the gradient of `f` at `x` using `fdm`. Assumes that `f(x)` is scalar.
 """
-function grad(fdm, f, x::Vector{T}) where T<:Real
+function grad(fdm, f, x::Vector{T}) where T<:Number
```

Comment:
I suppose really we'd need to add some tests on `Complex` cases if we're going to support them?

Reply:
Yes probably. Though it would be nice, if we could just do something analogous to JuliaDiff/ChainRulesCore.jl#28 and not having to write tests twice :wink: